modules.yml

```diff
@@ -1035,3 +1035,13 @@
   path: reporting/xml
   required_keys: []
   version: '1.1'
+- author: Cam Barts (@cam-barts)
```

Comment:
Please place this within the index at the proper location. They are sorted alphabetical by path. Also, if you run a full index and look at the output you'll can see where it lands.

Reply:
That's something I didn't know and just didn't really notice in passing, and the docs don't mention anything about index placement. I'll be sure to add appropriately